doc/manual/manual.bib

```diff
@@ -10601,27 +10682,30 @@ @MastersThesis{Sch95
 }
 
 
-@TechReport{Sch97a,
-  author =       {Frank Schmidt},
-  title =        {Construction of discrete transparent boundary conditions for
-                  {S}chr{\"o}dinger-type equations},
-  institution =  {Konrad-Zuse-Zentrum f{\"u}r Informationstechnik Berlin},
-  year =         1997,
-  type =         {Preprint SC 97-60}
+@article{Sch97a,
+author = {Schmidt, Frank},
+year = {2001},
+month = {09},
+pages = {},
+title = {Construction of discrete transparent boundary conditions for
+                {S}chr{\"o}dinger-type equations},
+volume = {9},
+journal = {Surv. Math. Ind.}
 }
 
 
-
 @TechReport{Sch97b,
   author =       {Frank Schmidt},
   title =        {Computation of discrete transparent boundary conditions
                   for the 2{D} {H}elmholtz equation},
   institution =  {Konrad-Zuse-Zentrum f{\"u}r Informationstechnik Berlin},
-  year =         1997,
-  type =         {Preprint SC 97-64}
+  year =         1998,
+  volume =      30,
+  pages =       427--4641,
+  doi =         https://doi.org/10.1023/A:1006919107753
 }
 
-
+-441
```

Comment:
what is this?

Reply:
artifact. i know who i am calling the next time I am looking for a needle in a haystack.